folio-service-tools-spring-dev/src/main/java/org/folio/spring/tools/kafka/KafkaAdminService.java

```diff
@@ -42,6 +46,16 @@ public void createTopics(String tenantId) {
     kafkaAdmin.initialize();
   }
 
+  public void deleteTopicsByTenant(String tenantId) {
```

Comment:
```suggestion
  public void deleteTopics(String tenantId) {
```
I suggest to be consistent with `createTopics`